src/lib/output/themes/DefaultTheme.ts

```diff
@@ -157,10 +157,12 @@ export class DefaultTheme extends Theme {
      * @returns        The root navigation item.
      */
     getNavigation(project: ProjectReflection): NavigationItem {
+        const multipleEntryPoints =
+            project.getReflectionsByKind(ReflectionKind.Module).length > 1;
         const builder = new NavigationBuilder(
             project,
             project,
-            this.application.options.getValue("entryPoints").length > 1
```

Comment:
"entryPoints" option no longer matches the number of modules we're documenting so I needed a change here. I thing using `project.getReflectionsByKind(ReflectionKind.Module).length` is valid, but that might not be the number ok modules we're going to actually document, so please let me know if this needs more investigation.

Reply:
This does change the logic a bit, and unfortunately is considerably more expensive since it loops over every reflection in the project (both direct children and their children). A better check would be `project.getChildrenByKind(ReflectionKind.Module).length !== 0`

Reply:
This seems to work :+1: